api/client/options.proto

```diff
@@ -3,7 +3,17 @@ syntax = "proto3";
 package nighthawk.client;
 
 import "google/protobuf/duration.proto";
+import "validate/validate.proto";
+import "envoy/api/v2/core/base.proto";
 
+
+message RequestOptions {
+  string request_method = 1;
```

Comment:
Should this be an enum? See envoy.api.v2.core.RequestMethod, you can probably reuse that.

Reply:
Done